helm/cluster-api/values.yaml

```
@@ -8,7 +8,7 @@ images:
     name: giantswarm/kubeadm-bootstrap-controller
   controlplane:
     name: giantswarm/kubeadm-control-plane-controller
-  tag: v1.4.5-gs-c73220ed8  # upstream v1.4.5 + https://github.com/kubernetes-sigs/cluster-api/pull/8586 (Makefile improvement)
+  tag: v1.4.7-gs-7030a8361  # upstream v1.4.7
```

```suggestion
  tag: v1.4.7-gs-7030a8361  # upstream v1.4.7 + https://github.com/kubernetes-sigs/cluster-api/pull/8586 (Makefile improvement)
```

since that change wasn't cherrypicked into v1.4.x yet and hence is still a custom change

---

Oh crap I thought they did 🤦🏻